codegenerator/cli/npm/envio/evm.schema.json

```diff
@@ -11,15 +11,9 @@
         "null"
       ]
     },
-    "contracts": {
-      "description": "Global contract definitions that must contain all definitions except addresses. You can share a single handler/abi/event definitions for contracts across multiple chains.",
-      "type": [
-        "array",
-        "null"
-      ],
-      "items": {
-        "$ref": "#/$defs/GlobalContract_for_ContractConfig"
-      }
+    "name": {
+      "description": "Name of the project",
+      "type": "string"
```

Comment:
Unfortunately I looked at the code for schema rs and it will always order the keyword "description" first for it's own objects so we basically just have a name collision there. I don't think it's very important to change right now. Not sure if we should even keep description in the schema.

Reply:
Hmmm, I think it's not used in any way.